monitoring/monitoring/documentation.go

```diff
@@ -3,73 +3,167 @@ package monitoring
 import (
 	"bytes"
 	"fmt"
+	"io"
+	"regexp"
 	"strings"
 )
 
-func renderDocumentation(containers []*Container) ([]byte, error) {
-	var b bytes.Buffer
-	fmt.Fprintf(&b, `# Alert solutions
+const (
+	canonicalAlertSolutionsURL = "https://docs.sourcegraph.com/admin/observability/alert_solutions"
+	canonicalDashboardsDocsURL = "https://docs.sourcegraph.com/admin/observability/dashboards"
+
+	alertSolutionsFile = "alert_solutions.md"
+	dashboardsDocsFile = "dashboards.md"
+)
+
+const alertSolutionsHeader = `# Alert solutions
+
+<!-- DO NOT EDIT: generated via: go generate ./monitoring -->
 
 This document contains possible solutions for when you find alerts are firing in Sourcegraph's monitoring.
-If your alert isn't mentioned here, or if the solution doesn't help, [contact us](mailto:support@sourcegraph.com)
-for assistance.
+If your alert isn't mentioned here, or if the solution doesn't help, [contact us](mailto:support@sourcegraph.com) for assistance.
+
+To learn more about Sourcegraph's alerting and how to set up alerts, see [our alerting guide](https://docs.sourcegraph.com/admin/observability/alerting).
 
-To learn more about Sourcegraph's alerting, see [our alerting documentation](https://docs.sourcegraph.com/admin/observability/alerting).
+`
+
+const dashboardsHeader = `# Dashboards reference
 
 <!-- DO NOT EDIT: generated via: go generate ./monitoring -->
 
-`)
+This document contains a complete reference on Sourcegraph's available dashboards, as well as details on how to interpret the panels and metrics.
+
+To learn more about Sourcegraph's metrics and how to view these dashboards, see [our metrics guide](https://docs.sourcegraph.com/admin/observability/metrics).
+
+`
+
+func fprintSubtitle(w io.Writer, text string) {
```

Comment:
```suggestion
func printSubtitle(w io.Writer, text string) {
```

fprint would suggest it is in the f family with a formatting string and varargs at the end for value substitution. similar for the others (this is a nit-pick, it's your choice :-) )

Reply:
I think formatting string is `Printf`, and writing to a writer is `Fprint`, and writing to a writer AND formatting string is `Fprintf` (quite a mouthful 😅 ), so I think this naming is correct? https://golang.org/pkg/fmt/#Fprintf

Reply:
you're right, sorry